package NYT.NObjectClient.NProto;

import "yt/core/misc/guid.proto";
import "yt/core/ytree/attributes.proto";

////////////////////////////////////////////////////////////////////////////////

message TReqCreateObject
{
    optional NYT.NProto.TGuid transaction_id = 1;
    required int32 type = 2;
    optional NYT.NYTree.NProto.TAttributes object_attributes = 3;
    optional string account = 4;

    extensions 100 to max;
}

message TRspCreateObject
{
    required NYT.NProto.TGuid object_id = 1;

    extensions 100 to max;
}

////////////////////////////////////////////////////////////////////////////////
